tools/plugins/com.liferay.ide.layouttpl.ui/src/com/liferay/ide/layouttpl/ui/wizard/NewLayoutTplWizardPage.java

```diff
@@ -111,6 +113,19 @@ protected void createTemplateInfoGroup( Composite parent )
 
         Button templateFileBrowse = SWTUtil.createPushButton( parent, Msgs.browse, null );
 
+        id.addModifyListener( new ModifyListener()
```

Comment:
This should not be done manually but instead done via the NewLayoutTplDataModelProvider.validate method.